saleor/graphql/account/types.py

```diff
@@ -365,6 +369,10 @@ def __resolve_reference(root, _info, **_kwargs):
             return graphene.Node.get_node_from_global_id(_info, root.id)
         return get_user_model().objects.get(email=root.email)
 
+    @staticmethod
+    def resolve_language_code(root, _info, **_kwargs):
+        return LanguageCodeEnum[str_to_enum(root.language_code)]
```

Comment:
What if provided language code is missing in `LanguageCodeEnum`, that will throw `KeyError`?
If yes maybe we should use `get` method on `LanguageCodeEnum`

Reply:
You set the language_code over API, you use for this the same enum, so there is no possibility to set missing code.  user.language_code cannot be empty and as a default value it receives an EN, so I don't think that we should worry here